experiments/xls/measurement_xls_importer.py

```diff
@@ -33,3 +33,15 @@ def import_measurements(self) -> None:
                 logger.error(f"Failed to import measurement with row number {i+1}")
                 logger.error(e)
                 traceback.print_exc()
+
+    def delete_measurements(self):
```

Comment:
Should this invoke a cleanup of Slides/Samples not referenced by any measurement afterwards?

Reply:
I don't think it's necessary at this stage since things may start being different when data is pulled from FreezerPro. For abandoned objects, we can have a maintenance cleanup script to run from time to time so that the speed of deletion doesn't increase.